lib/class-factory.js

```diff
@@ -669,9 +669,12 @@ function ClassFactory (vm) {
           try {
             const fieldName = invokeObjectMethodNoArgs(env.handle, field, fieldGetName);
             try {
-              const fieldjsName = env.stringFromJni(fieldName);
+              let fieldjsName = env.stringFromJni(fieldName);
               const fieldHandle = env.newGlobalRef(field);
               fieldHandles.push(fieldHandle);
+              while (jsMethods.hasOwnProperty(fieldjsName)){
```

Comment:
- Let's move these 3 lines to right after the `let`, as they belong together. Then put a blank line between that and the `fieldHandle` stuff (those two lines), and then one blank line before assigning to `jsFields`. Just to make the code a bit more readable.
- Missing a space before `{`.